modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -1482,6 +1709,19 @@ private void addEmailDomainsToOrganization(String organizationId, String... doma
         validateHttpStatusCode(response, HttpStatus.SC_CREATED);
     }
 
+    private String getMetaAttributesEndpoint(boolean isForward, int limit, String after, String before) {
+
+        if (isForward) {
+            return ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_META_ATTRIBUTES_API_PATH + QUESTION_MARK +
+                    LIMIT_QUERY_PARAM + EQUAL + limit + AMPERSAND + RECURSIVE_QUERY_PARAM + EQUAL + false +
+                    (after != null ? AMPERSAND + AFTER_QUERY_PARAM + EQUAL + after : "");
```

Comment:
Can we add the base endpoint into a common variable, instead of having it repeated? 

```
String baseEndpoint = ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_META_ATTRIBUTES_API_PATH + 
                          QUESTION_MARK + LIMIT_QUERY_PARAM + EQUAL + limit + 
                          AMPERSAND + RECURSIVE_QUERY_PARAM + EQUAL + false;
if (isForward) {
    return baseEndpoint + (after != null ? AMPERSAND + AFTER_QUERY_PARAM + EQUAL + after : "");
} else {
     return baseEndpoint + (before != null ? AMPERSAND + BEFORE_QUERY_PARAM + EQUAL + before : "");
}
```

Reply:
Addressed with https://github.com/wso2/product-is/pull/21054/commits/d1921f987aae0800b4ba25ddc21c7274bca95774